bundle/edu.gemini.spModel.io/src/main/scala/edu/gemini/spModel/io/impl/migration/to2015B/To2015B.scala

```diff
@@ -116,6 +194,36 @@ object To2015B {
     ps
 
   }
+
+  // Proper Motion conversion to degrees/yr
```

Comment:
Yep, changed these to `Angle` and parallax to `Option[Angle]` in b529845d4a2b88de551a18990cda9e4515a8b406.